README.md

```diff
@@ -114,3 +114,14 @@ View the query you made in Grafana by clicking the **View query** button.
 ![View Grafana query](https://github.com/lightstep/lightstep-metrics-datasource/raw/main/images/docs/graf_metciViewQuery.png)
 
 <!-- TODO: add a link to the Learning Path for Prom+Grafana, once available -->
+
+### Running on docker for development and testing
```

Comment:
could we add this in either a separate 'Development' section, or putting it alongside the other installation instructions?

Reply:
Moved up to the installation instructions part 👍